crates/router/src/core/payouts/helpers.rs

```diff
@@ -1389,3 +1392,14 @@ pub async fn get_additional_payout_data(
         }
     }
 }
+
+pub fn get_locale_from_header(headers: Option<&HeaderMap>) -> String {
```

Comment:
Also, by the looks of it, the `headers` param need not be an `Option`, at least for the payouts module? The existing function accepted it as a required param.